boards/avsextrem/board_init.c

```diff
@@ -41,19 +41,21 @@ void init_clks1(void)
     PLLCON &= ~0x0002;
     pllfeed();
 
-    while (PLLSTAT & BIT25);  /* wait until PLL is disconnected before
-                               * disabling - deadlock otherwise */
+    /* wait until PLL is disconnected before disabling - deadlock otherwise */
+    while (PLLSTAT & BIT25) {} 
 
     // Disable PLL
     PLLCON &= ~0x0001;
     pllfeed();
 
-    while (PLLSTAT & BIT24); // wait until PLL is disabled
+    // wait until PLL is disabled
```

Comment:
I guess you/we'll fix **all** C++ style comments in this file in a separate PR, right?

Reply:
that's the idea :P